test/helpers.bash

```diff
@@ -644,7 +645,7 @@ function run_nc_test() {
     fi
 
     nsenter -n -t "${CONTAINER_NS_PIDS[$container_ns]}" timeout --foreground -v --kill=10 5 \
-        nc $nc_common_args -l -p $container_port &>"$NETAVARK_TMPDIR/nc-out" <$stdin &
+        nc $nc_common_args -l -p $container_port &>"$NETAVARK_TMPDIR/nc-out" <&$stdin &
```

Comment:
Although CI is passing, it's not for the reasons described in your commit: `$stdin` is, at this point, undefined. I expected bash to treat this as an error, but it doesn't, and I'm completely lost as to what exactly is happening.

I *think* the answer here is to remove `<&$stdin` entirely, given that you've set up the `exec`, but I'm sorry I can't approve this as it is.

Reply:
sorry I don't follow why do you think $stdin is undefined?

Reply:
I will try to remove it, if that makes it I happily do that as it makes the test code simpler

Reply:
no wait hold on; I'm refreshing my memory on `{varname}`